### ISSUE TITLE
Rewrite with `elm-watch`

### ISSUE THREAD

**Issue by kutyel** [jgrenat/elm-workshop#9]
Hey! This is a really nice workshop and I love how the tests and inputs are constructed, wouldn't it be awesome if we use [`elm-watch`](https://github.com/lydell/elm-watch) for hot reloading? I wonder if this is compatible with `elm reactor` and we can mix the best of both worlds! 🤔

**Comment by jgrenat** [comment]
Hey! Sorry, I've just noticed your message! The idea was to avoid having too many things to install for those who want to try Elm, because making them install `elm-watch` means also that they need `npm` installed

I wanted to try gitpod for an installation-free start, in which case it may be possible to have `elm-watch` too.
What do you think?

**Comment by kutyel** [comment]
Yeah, maybe having a browser-only version of the workshop might be a good idea, hot-reloading and a web browser is all most people need nowadays 😄
